### PR TITLE
feat: added tooltip to display full cumulative funding value

### PR DESCRIPTION
**Pull request by Clish254**
## Description
 Also updated funding payment to display numbers greater than 0.0001 and < 0.0001 for numbers less than that.
## Types of changes

- [ ] Technical Debt (non-breaking change which removes unused code/assets)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] The related ClickUp task has been linked to this PR
- [ ] The person creating the pull request is listed in "Assignees"
- [ ] Change requires updated documentation

### CODE REVIEW

File: src/pages/Account/components/FundingHistory.tsx [hunk 1]
```diff
@@ -213,15 +238,21 @@ const FundingHistory: FC = () => {
         <ACCOUNTVALUESCONTAINER>
           <ACCOUNTVALUE>
             <p>Cumulative Funding:</p>
-            <p>
+            <div className="tooltip">
```
Comment (deadeagle63) [inline src/pages/Account/components/FundingHistory.tsx]: We have a Tooltip component already, should we use that instead?

File: src/pages/Account/components/Mobile/FundingHistory.tsx [hunk 2]
```diff
@@ -199,15 +225,21 @@ const MobileFundingHistory: FC = () => {
         <ACCOUNTVALUESCONTAINER>
           <ACCOUNTVALUE>
             <p>Cumulative Funding:</p>
-            <p>
+            <div className="tooltip">
```
Comment (deadeagle63) [inline src/pages/Account/components/Mobile/FundingHistory.tsx]: Same as above.

**Review by deadeagle63 — CHANGES_REQUESTED** [review]

File: src/pages/Account/components/FundingHistory.tsx (hunk 1 above)
Comment (Clish254) [reply src/pages/Account/components/FundingHistory.tsx]: > We have a Tooltip component already, should we use that instead?

I think we could but that one has an info icon and it takes text that is supposed to be a description but for this feature it's just a tooltip displayed when the cumulative funding number is hovered on to display all the decimal places.

Comment (deadeagle63) [reply src/pages/Account/components/FundingHistory.tsx]: The tooltip has a built in prop to remove the icon. I believe it is infoIcon (boolean which defaults to true)

Comment (walt-1) [reply src/pages/Account/components/FundingHistory.tsx]: yes, and feel free to adjust the component to fit the usecase. there will likely be other similar use cases that could benefit from the update

Comment (Clish254) [reply src/pages/Account/components/FundingHistory.tsx]: Managed to use title and children props to fit this usecase, i've pushed the commit to this pr.

**Review by deadeagle63 — APPROVED** [review]

**Review by walt-1 — APPROVED** [review]

**Pull request merged**